server/service/handler.go

```diff
@@ -551,6 +556,7 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	neMDM.GET(apple_mdm.EnrollPath, mdmAppleEnrollEndpoint, mdmAppleEnrollRequest{})
 	neMDM.GET(apple_mdm.InstallerPath, mdmAppleGetInstallerEndpoint, mdmAppleGetInstallerRequest{})
 	neMDM.HEAD(apple_mdm.InstallerPath, mdmAppleHeadInstallerEndpoint, mdmAppleHeadInstallerRequest{})
+	neMDM.GET("/api/_version_/fleet/mdm/apple/bootstrap", downloadBootstrapPackageEndpoint, downloadBootstrapPackageRequest{})
```

Comment:
@lukeheath I wanted to bring your attention to these endpoints, do they look good to you? A couple of notes:

- `GET /api/_version_/fleet/mdm/apple/bootstrap/{team_id:[0-9]+}/metadata` is not specified in #10934 but I think it's needed in the UI too (to show the package name and the token to download)
- `GET /api/_version_/fleet/mdm/apple/bootstrap` receives a `token=?` query parameter

Reply:
Thanks for the heads up; I'll update the relevant tickets.

Reply:
Is this part of the API or is it more a device/host related endpoint? (`/api/_version_/fleet/` prefix)

Reply:
that's a good point, I know @lukeheath has thoughts about using version vs not, I'll change it with whatever you folks decide.

Reply:
We'll likely call this endpoint from the frontend, so let's keep it where it is (`/fleet/mdm/apple/bootstrap`). Thanks!